azure-pipelines-template.yml

```diff
@@ -42,10 +42,9 @@ jobs:
   - script: npm run prebuild
     displayName: Test prebuild
 
-  - ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
-    - task: PublishBuildArtifacts@1
-      displayName: Publish Artifacts
-      condition: and(succeeded(), eq(variables['publish_artifacts'], 'true'))
-      inputs:
-        artifactName: prebuilds
-        pathtoPublish: '$(System.DefaultWorkingDirectory)/prebuilds'
+  - task: PublishBuildArtifacts@1
```

Comment:
Not entirely sure if I need to change anything in the azure pipeline for it to deploy the folder to npm?

cc @mischnic any ideas? I barely know how this works